File: Lab3/src/model/Result.java
package model;

import com.google.gson.annotations.SerializedName;

public class Result {
    @SerializedName("message")
    private String message;

    @SerializedName("account")
    private Account account;

    @SerializedName("realNumber")
    private int realNumber;

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public Account getAccount() {
        return account;
    }

    public void setAccount(Account account) {
        this.account = account;
    }

    public int getRealNumber() {
        return realNumber;
    }

    public void setRealNumber(int realNumber) {
        this.realNumber = realNumber;
    }

    @Override
    public String toString() {
        return String.format("{[message]: %s; [account]: %s; [realNumber]: %s}", message, account, realNumber);
    }
}

File: Lab1/Lab1/Lab1/Utils/Result.cs
namespace Lab1
{
    class Result
    {
        public string Key { get; set; }
        public string Value { get; set; }

        public Result(string key, string value)
        {
            Key = key;
            Value = value;
        }

        public override string ToString()
        {
            return $"[{Key}]: {Value}";
        }
    }
}
